applications/FluidDynamicsApplication/custom_strategies/schemes/velocity_bossak_sensitivity_builder_scheme.h

```diff
@@ -0,0 +1,230 @@
+//    |  /           |
```

Comment:
Why not storing the sensitivity builders in the utilities as it is done in the Kratos core base one?